c_glib/arrow-flight-glib/client.cpp

```diff
@@ -251,12 +251,12 @@ gaflight_client_new(GAFlightLocation *location,
   arrow::Status status;
   if (options) {
     const auto flight_options = gaflight_client_options_get_raw(options);
-    status = arrow::flight::FlightClient::Connect(*flight_location,
-                                                  *flight_options,
-                                                  &flight_client);
+     arrow::Result<std::unique_ptr<arrow::flight::FlightClient>> result = arrow::flight::FlightClient::Connect(*flight_location,
+                                                        *flight_options);
+    status = std::move(result).Value(&flight_client);
```

Comment:
A few things:

There's an extra space at the beginning of the line.

Why not use `auto` as before and keep things aligned, in a consistent style with everything else?

Reply:
GLib doesn't use an auto-formatter, unfortunately. So, sorry for the nit-picking here :(

@kou have we considered automated formatting for the GLib code?

Reply:
We have considered it a few times but we didn't do any actions for it.
It may be a good time to work on it: https://issues.apache.org/jira/browse/ARROW-16137